### PR TITLE
Test CI

### PR DESCRIPTION
**Pull request by thehoait**

### CODE REVIEW

File: app/src/main/java/com/example/hoaht/myapplication/MainActivity.java [hunk 1]
```diff
@@ -8,6 +8,11 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        Test();
         setContentView(R.layout.activity_main);
     }
+
+    private void Test() {
```
Comment (thehoait) [inline app/src/main/java/com/example/hoaht/myapplication/MainActivity.java]: ERROR: Name 'Test' must match pattern '^[a-z]([_a-zA-Z0-9]+)*$'.

Comment (thehoait) [inline app/src/main/java/com/example/hoaht/myapplication/MainActivity.java]: ERROR: [MethodNamingConventions] Method names should not start with capital letters
http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/naming.html#MethodNamingConventions

File: app/src/main/java/com/example/hoaht/myapplication/MainActivity.java [hunk 2]
```diff
@@ -8,6 +8,15 @@
...
+        // TODO: implement later
+    }
+
+    private void Test2() {
```
Comment (hoaht-support) [inline app/src/main/java/com/example/hoaht/myapplication/MainActivity.java]: ERROR: Name 'Test2' must match pattern '^[a-z]([_a-zA-Z0-9]+)*$'.

Comment (hoaht-support) [inline app/src/main/java/com/example/hoaht/myapplication/MainActivity.java]: ERROR: [MethodNamingConventions] Method names should not start with capital letters
http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/naming.html#MethodNamingConventions

File: app/src/main/java/com/example/hoaht/myapplication/MainActivity.java [hunk 3]
```diff
@@ -8,6 +8,15 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        Test();
         setContentView(R.layout.activity_main);
     }
+
+    private void Test() {
```
Comment (hoaht-support) [inline app/src/main/java/com/example/hoaht/myapplication/MainActivity.java]: ERROR: [MethodNamingConventions] Method names should not start with capital letters
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#MethodNamingConventions

File: app/src/main/java/com/example/hoaht/myapplication/MainActivity.java (hunk 2 above)
Comment (hoaht-support) [inline app/src/main/java/com/example/hoaht/myapplication/MainActivity.java]: ERROR: [MethodNamingConventions] Method names should not start with capital letters
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#MethodNamingConventions

**Pull request merged**